dev/breeze/src/airflow_breeze/breeze.py

```diff
@@ -265,6 +265,9 @@
                 ],
             },
         ],
+        "breeze exec": [
```

Comment:
I tihnk this logically goes after `start-airflow` and before `stop`  command so you should move it up a bit.